subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/ui/SubscriptionWebViewViewModel.kt

```diff
@@ -299,8 +299,8 @@ class SubscriptionWebViewViewModel @Inject constructor(
         }
     }
 
-    private fun isFreeTrialEligible(): Boolean {
-        return privacyProFeature.privacyProFreeTrialJan25().isEnabled(Cohorts.TREATMENT)
+    private suspend fun isFreeTrialEligible(): Boolean {
+        return privacyProFeature.privacyProFreeTrialJan25().isEnrolledAndEnabled(Cohorts.TREATMENT)
```

Comment:
@aitorvs I removed the Free Trial experiment and implemented the feature for all users, so you'll see some conflicts now. Feel free to ping me if you have any questions, but I don’t think you’ll need to make any further changes in Subscriptions as the experiment is no longer in place.